operator/config/default/webhookcainjection_patch.yaml

```diff
@@ -4,8 +4,6 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
```

Comment:
just a nitpick, but should we rename this file, since it's purpose is not about injecting the cert-manager annotation anymore

Reply:
good idea!